wagtailcaptcha/models.py

```diff
@@ -18,14 +18,10 @@ def __init__(self, *args, **kwargs):
         self.form_builder = WagtailCaptchaFormBuilder
 
     def process_form_submission(self, form):
-        super(AbstractEmailForm, self).process_form_submission(form)
-
-        if self.to_address:
-            content = ''
-            for x in form.fields.items():
-                if not isinstance(x[1], ReCaptchaField):  # exclude ReCaptchaField from notification
-                    content += '\n' + x[1].label + ': ' + text_type(form.data.get(x[0]))
-            send_mail(self.subject, content, [self.to_address], self.from_address,)
+        if WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME in form.fields.keys():
+            form.fields.pop(WagtailCaptchaFormBuilder.CAPTCHA_FIELD_NAME)
+
+        return super(WagtailCaptchaEmailForm, self).process_form_submission(form)
```

Comment:
So that the captcha doesn't propagate to database, please also add this exact same function to WagtailCaptchaForm.